data21/focuses.json

```diff
@@ -196,7 +196,7 @@
         "url": "allModules/38945_M_WE2.json"
       },
       {
-        "id": "WIoT",
+        "id": "WsoT",
         "name": "Wireless and IoT",
         "url": "allModules/37157_M_WIoT.json"
```

Comment:
Why rename the id, if the id on Adunis did not change? They normally change the url as well, no?

Reply:
Are you unifying 2 modules? If so, which ones?

Reply:
I wonder, should we change to url to STD23 version as well?
It is more accurate to keep the STD21 url.
But also a slight discrepancy in the data, since the entry in modules.json references a different url...